README.md

```diff
@@ -211,7 +211,7 @@ color:
 # This specifies the date format for the date column. The freeform format
 # accepts an strftime like string.
 # When "classic" is set, this is set to "date".
-# Possible values: date, relative, +<date_format>
+# Possible values: date, relative, '+<strftime_date_format>'
```

Comment:
A small example of such value would be useful. `date: ?`

Reply:
@alexxbb That would help. 
@robole I think it would be better to keep it as `date_format` itself (we use the same name everywhere else). We can probably just mention what it is separately with an example.

Reply:
I updated the text @alexbb @meain 

```
# Possible values: date, relative, '+<date_format>' e.g. date: '+%d %b %y %X' will give you 
# a date like this: 17 Jun 21 20:14:55
```